src/main/java/seedu/address/storage/SecurityUtil.java

```diff
@@ -0,0 +1,119 @@
+package seedu.address.storage;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.security.GeneralSecurityException;
+import java.security.Key;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.spec.SecretKeySpec;
+
+import seedu.address.MainApp;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Contains utility methods used for encrypting and decrypting files for Storage
+ */
+public class SecurityUtil {
+    private static final Logger logger = LogsCenter.getLogger(MainApp.class);
+    private static String password = new String("test");
```

Comment:
Interesting implementation. So this means that anyone with access to the codebase can decrypt any addressbook data right?